pyhomematic/devicetypes/thermostats.py

```diff
@@ -162,13 +162,29 @@ def __init__(self, device_description, proxy, resolveparamsets=False):
         super().__init__(device_description, proxy, resolveparamsets)
 
         # init metadata
-        self.SENSORNODE.update({"ACTUAL_TEMPERATURE": [1]})
+        self.SENSORNODE.update({"ACTUAL_TEMPERATURE": [1],
+                                "VALVE_STATE": [1]})
```

Comment:
I agree, that we don't need an extra entity in HASS caused by the added sensor node.
However, the possibility to handle the VALVE_STATE (and other attributes from any device for that matter) through a template sensor seems to be a great addition for the HASS documentation.